### DISCUSSION TITLE
cache-path seems to be ignored

### DISCUSSION THREAD

**Discussion by shabbychef** [quarto-dev/quarto-cli#534] [Q&A] [answered]
(First wanted to confirm this is the right forum for quarto Q&A, and not, say stackoverflow.)

I am making an HTML doc. I have the following in my document metadata:
```
cache: true
cache-path: cache/autorisk_
```
However, the `cache-path` seems to be ignored, and instead the cache is put in `autorisk_cache/html/`. Must I enable `cache-path` per execution context? Or should it be under `execute:`?

**Comment by jjallaire** [comment, marked as answer]
You are correct, that is a knitr option that actually doesn't survive the normal Rmd/Qmd pipeline. For now I will remove the option from the docs (but I'll transfer this to issues and leave it open as a reminder to try to make this work).

**Reply by shabbychef** [reply]
I see, thanks!
